pylearn2/datasets/svhn.py

```diff
@@ -29,14 +30,28 @@ class SVHN(dense_design_matrix.DenseDesignMatrixPyTables):
 
     Parameters
     ----------
-    which_set : WRITEME
-    path : WRITEME
-    center : WRITEME
-    scale : WRITEME
-    start : WRITEME
-    stop : WRITEME
-    axes : WRITEME
-    preprocessor : WRITEME
+    which_set : str
+                Subset name.
```

Comment:
Indents should be 4 spaces according to the docstring standard. Also a list of valid values here (I assume 'train', 'test', 'valid', but aren't there some weird ones?).